src/icons/clone-icon.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where did you get this clone icon from and what is the icon license?

Reply:
@sadakchap Just pinging in case you missed this comment. We need to be mind the license of the icon, once this is resolved, this PR is ready for merge.

Reply:
I got this icon from font awesome, [License](https://fontawesome.com/license/free) we are using solid version of their icon

Reply:
Sure